config.js

```diff
@@ -28,12 +28,12 @@ var config = module.exports.config = {};
  * @param  {String}    path            The path to the UI static assets
  */
 config.ui = {
-    'path': '../3akai-ux'
+    'path': './3akai-ux'
```

Comment:
Try a "git grep 3akai-ux" in puppet-hilary, at first glance it seems likely to cause some problems.
Presumably this change moves 3akai-ux into the Hilary dir.